generators/generator-base-private.js

```diff
@@ -54,7 +53,6 @@ module.exports = class extends Generator {
         this.env.options.appPath = this.config.get('appPath') || CLIENT_MAIN_SRC_DIR;
         // expose lodash to templates
         this._ = _;
-        this.createConfigFromNewConfFile();
```

Comment:
No longer required after changes to importJdl and also this was causing the config to be not written during regeneration